sdk/datamigration/Microsoft.Azure.Management.DataMigration/src/Generated/Models/MongoDbCollectionProgress.cs

```diff
@@ -66,8 +66,8 @@ public MongoDbCollectionProgress()
         /// object. For a collection, this is the database-qualified name. For
         /// a database, this is the database name. For the overall migration,
         /// this is null.</param>
-        public MongoDbCollectionProgress(long bytesCopied, long documentsCopied, string elapsedTime, IDictionary<string, MongoDbError> errors, long eventsPending, long eventsReplayed, string resultType, string state, long totalBytes, long totalDocuments, System.DateTimeOffset? lastEventTime = default(System.DateTimeOffset?), System.DateTimeOffset? lastReplayTime = default(System.DateTimeOffset?), string name = default(string), string qualifiedName = default(string))
-            : base(bytesCopied, documentsCopied, elapsedTime, errors, eventsPending, eventsReplayed, resultType, state, totalBytes, totalDocuments, lastEventTime, lastReplayTime, name, qualifiedName)
+        public MongoDbCollectionProgress(long bytesCopied, long documentsCopied, string elapsedTime, IDictionary<string, MongoDbError> errors, long eventsPending, long eventsReplayed, string state, long totalBytes, long totalDocuments, System.DateTimeOffset? lastEventTime = default(System.DateTimeOffset?), System.DateTimeOffset? lastReplayTime = default(System.DateTimeOffset?), string name = default(string), string qualifiedName = default(string))
```

Comment:
This results in a breaking change since we are removing a parameter from a public method.

The spec PR does not have the Approved-BreakingChanges label please work with the breaking change board to get this label added.